README.md

```diff
@@ -1,56 +1,49 @@
- ```shell 
+## Setting up Gatsby_code_profiles
+
+To set up the Gatsby code profile, follow these steps:
+
+ 1. Clone the repository **into your computer's root directory**.
+
+ ```shell
+ cd ~/
  git clone https://github.com/Ekwuno/gatsby_code_profiles.git
  ```
- Then change directory into the code profile.
+ Then change directories into the code profile.
  ```shell 
- git clone cd gatsby_code_profiles 
+ cd gatsby_code_profiles 
  ```
- In gatsby_code_profiles directory exists the presentation folder which houses the custom settings. 
- 
- > Multiple sub folders can exist for code profiles and you would have to change directory to access the setting.
+ In the `gatsby_code_profiles` directory exists the presentation folder which houses the custom settings.  
  
- 
- 2. Set up alias
-
-Aliases are helpful for shortening commands and also help to reduce the cognitive load of learning a really long command and having to type it everytime. Vscode has a command line alias to [launch from the command line](https://code.visualstudio.com/docs/setup/mac#_launching-from-the-command-line) and this comes 
-in handy.
-
-For example, this repo lives in ~/gatsby_code_profiles and the command to launch my presentation settings and extensions is:
-
-```code --extensions-dir ~/gatsby_code_profiles/presentation/exts --user-data-dir ~/gatsby_code_profiles/presentation/data```
+ 2. Set up an alias
 
-The alias in the .zshrc file looks like this :
+To prevent from having to type a lengthy launch command every time, you'll want to set up an alias for your Terminal to launch VS Code with these settings and extensions from `~/gatsby_code_profiles`.
 
-`alias gatsby-teach="code --extensions-dir ~/gatsby_code_profiles/presentation/exts --user-data-dir ~/gatsby_code_profiles/presentation/data"
+To set an alias for this command using [Oh My Zsh](https://ohmyz.sh/), paste this command into your `.zshrc` file:
 
-> Now anytime you want to lunch VScode with the custom settings all you have to do it run gatsby-teach {name of directory}
+```alias gatsby-teach="code --extensions-dir ~/gatsby_code_profiles/presentation/exts --user-data-dir ~/gatsby_code_profiles/presentation/data"```
 
-# Lunching Profile
+> Now anytime you want to launch VS Code with the custom settings, you can run `gatsby-teach {name of directory}`.
 
+## Launching the profile
 
-Restart your terminal and test out the command. You can include a project path to open it with that profile , or leave the directory blank to start a new project:
-`gatsby-teach ./some-demo-directory`
+Restart your terminal and test out the command. You can include a project path to open it with that profile, or leave the directory blank to start a new project:
 
+```gatsby-teach ./some-demo-directory```
 
-### More profiles
+## Adding more profiles
 
-As more profiles are added the Name of profile will aslo be updated 
+As more VS Code profiles are added, the `Name of profile` will also be updated:
```

Comment:
This paragraph could use a little more explanation. I debated on suggesting we remove it, but figured more details would help!